imports/plugins/core/layout/client/templates/layout/alerts/reactionAlerts.js

```diff
@@ -70,7 +70,7 @@ Object.assign(Alerts, {
         if (isConfirm === true && typeof messageOrCallback === "function") {
           messageOrCallback(isConfirm);
         }
-      });
+      }).catch(_.noop);
```

Comment:
Good thoughts. I've updated this code to catch/silence the three usual sweet alert errors and re-throw anything else.